src/targets/services/installSuggestion.js

```diff
@@ -0,0 +1,42 @@
+import fetch from 'node-fetch'
+import CozyClient from 'cozy-client'
+
+global.fetch = fetch
+
+const schema = {
+  suggestion: {
+    doctype: 'io.cozy.suggestions'
+  },
+  konnectors: {
+    doctype: 'io.cozy.konnectors'
+  }
+}
+
+let suggestion = {}
+try {
+  suggestion = JSON.parse(process.env.COZY_COUCH_DOC)
+} catch (e) {
+  throw new Error(`Wrong formatted suggestion doc: ${e}`)
+}
+
+const client = new CozyClient({
+  uri: process.env.COZY_URL.trim(),
+  schema,
+  token: process.env.COZY_CREDENTIALS.trim()
+}).getStackClient()
+
+async function install(suggestion) {
+  if (suggestion.silenced) return // silenced, so not concerned
+  const slug = suggestion.slug
+  try {
+    await client.fetchJSON(
+      'POST',
+      `/konnectors/${slug}?Source=registry://${slug}/stable`
+    )
+  } catch (e) {
+    if (e.status === 409) return
+    throw new Error(`Error when installing ${slug}: ${e}`)
+  }
+}
+
+install(suggestion)
```

Comment:
```
const handleErr = err => {
  log('error', err)
}

try {
  install(suggestion).catch(handleErr)
} catch (err) {
  handleErr(err)
}
```

Reply:
I don't have any logger for now, so the error handling should be improved but I don't really know how the worker logging system is working for now